### DISCUSSION TITLE
Desert Sync and Tailscale

### DISCUSSION THREAD

**Discussion by saucedge** [NightscoutFoundation/xDrip#3602] [General]
While dealing with the whole firebase situation, it occurred to me that I should be able to use Tailscale to enable a follower phone to sync with my collector phone via my tailnet.  My understanding is that Desert Sync enables intranet-only phone-to-phone sync for those cases when the phones are on the same LAN but don't have internet access.  It seems like Desert Sync and Tailscale should work perfectly together and enable me to avoid needing firebase ever again.  The problem is, when I enter the Tailscale IP of the master into the follower's Desert Sync settings (100.x.x.x), it quickly reverts to the LAN subnet IP (192.168.x.x).  The 192.168.x.x IP of course fails to connect as soon as either phone leaves the home network.  Would it be possible to force the Desert Sync IPs to stick to the 100.x.x.x Tailscale addresses so the phone-to-phone sync would work wherever the phones might be?

Thanks for an amazing project!

**Comment by Navid200** [comment]
Does xDrip sync not work for you using the latest Nightly?

**Reply by saucedge** [reply]
I'm running 2024.04.19 which I think is the latest Beta?  I've always been hesitant to hop on nightly builds in general and even more so right now since I just transitioned from G6 to G7 and want to build some confidence before changing things I might not have to.  But, perhaps I should just go for it.  Being able to leverage Tailscale would still be a nice backup option for the future, though.  Thanks!

**Reply by Navid200** [reply]
xDrip sync was fixed in the last Nightly (July 23, 2024).
You need to update both the master and follower.
Please update both and let us know if there are any issues.

**Reply by saucedge** [reply]
Installed that nightly build and confirmed the regular sync seems to work without issue.  Thanks!

**Reply by Navid200** [reply]
Thanks for the feedback
